src/dash-table/reconcile/null.ts

```diff
@@ -0,0 +1,20 @@
+import { ITypeColumn } from 'dash-table/components/Table/props';
+import { IReconciliation } from '.';
+
+export const reconcileNull = (
+    value: any,
+    options: ITypeColumn | undefined
+): IReconciliation => {
+    const allowNull = Boolean(options && options.validation && options.validation.allow_null);
+    const nully = isNully(value);
+
+    return {
+        success: nully && allowNull,
+        value: nully ? null : value
```

Comment:
`reconcileNull` is very nice! Especially collapsing all nully values to actual `null` here 🏆